ajax/libs/chai-as-promised/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "chai-as-promised",
+  "version": "6.0.0",
+  "filename": "chai-as-promised.min.js",
+  "description": "Extends Chai with assertions about promises.",
+  "keywords": [
+    "chai",
+    "chai-plugin",
+    "browser",
+    "async",
+    "testing",
+    "assertions",
+    "promises",
+    "promises-aplus"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/domenic/chai-as-promised"
+  },
```

Comment:
@sufuf3 Do we need to do that? Because, the URL means the same thing.

@PeterDaveHello What do you think of coming up with a standard way to write repository URL's across all package.json files on cdnjs? Because every library uses a different way of writing it, we can be uniform in that way.